src/Fable.Arch/Fable.Arch.Html.fs

```diff
@@ -205,6 +205,15 @@ module Attributes =
             |> String.concat " "
             |> Class
 
+    /// Helper to build space separated class with a static part
+    let classBaseList b (list: (string*bool) seq) =
+        list
+            |> Seq.filter (fun (c,cond) -> cond)
+            |> Seq.map (fun (c, cond) -> c)
+            |> String.concat " "
+            |> sprintf "%s %s" b
+            |> Class
```

Comment:
`Class` with the capital letter can be strange at first, because it's make a special cased for this attribute where all the others are in camelCase.

Yes `class` is a reserved work. We could go with:
- className
- Class
- class'
- \`class\` (hard to write)
- classy

Or we could even go with custom operator for less code but it's probably a little to strong has a change ^^.

*Some Javascript libraries like the work `classy` but it's more for OOP approach but here we got `class` us.*

Here I would have a preference for `classy`because it's close to class and also it's simple to write. I find `className`a little too long but if you prefer to go with this one we could. Choice is yours between this too.